src/main/resources/default.properties

```diff
@@ -7,7 +7,7 @@ hibernate.hbm2ddl.auto=update
 jdbc.driverClassName=com.mysql.jdbc.Driver
 jdbc.url=jdbc:mysql://localhost:3306/expercise?useUnicode=true&characterEncoding=utf-8
 jdbc.user=root
-jdbc.password=
+jdbc.password=bakery
```

Comment:
Actually default is empty. For environment specific values it is possible to create environmentSpecific.properties. (It will be ignored by Git.)